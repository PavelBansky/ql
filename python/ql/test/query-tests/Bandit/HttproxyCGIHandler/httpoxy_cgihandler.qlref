Bandit/httpoxy_cgihandler.ql