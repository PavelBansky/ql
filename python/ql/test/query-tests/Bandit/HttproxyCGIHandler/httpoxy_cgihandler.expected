| Test.py:10:5:10:31 | Attribute | Consider possible security implications associated with wsgiref.handlers.CGIHandler module. |
